style/pi-hole.css

```diff
@@ -116,130 +155,135 @@
 }
 
 #resetButton {
-	color: #f00;
-	font-weight: 700;
+  color: #f00;
+  font-weight: 700;
 }
 
 .vertical-alignment-helper {
-    display: table;
-    width: 100%;
-    height: 100%;
-    pointer-events: none;
+  display: table;
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
 }
+
 .vertical-alignment-helper > .vertical-align-center {
-    display: table-cell;
-    vertical-align: middle;
+  display: table-cell;
+  vertical-align: middle;
 }
+
 .vertical-alignment-helper > .vertical-align-center > .modal-content {
-    width: 250px;
-    margin-left: auto;
-    margin-right: auto;
-    word-wrap: break-word;
-    pointer-events: all;
+  width: 250px;
+  margin-left: auto;
+  margin-right: auto;
+  word-wrap: break-word;
+  pointer-events: all;
 }
 
 .alSpinner {
-    top: 0.1em;
-    left: 0.1em;
-    width: 0.8em;
-    height: 0.8em;
-    border-radius: 50%;
-    border: 4px solid #c0c0c0;
-    border-right-color: transparent;
-    -webkit-animation: fa-spin 1s infinite linear;
-    -o-animation: fa-spin 1s infinite linear;
-    animation: fa-spin 1s infinite linear;
+  top: .1em;
+  left: .1em;
+  width: .8em;
+  height: .8em;
+  border-radius: 50%;
+  border: 4px solid #c0c0c0;
+  border-right-color: transparent;
+  -webkit-animation: fa-spin 1s infinite linear;
+  -o-animation: fa-spin 1s infinite linear;
+  animation: fa-spin 1s infinite linear;
 }
 
-.extratooltipcanvas{
-    -moz-user-select: none;
-    -webkit-user-select: none;
-    -ms-user-select: none;
-    user-select: none
+.extratooltipcanvas {
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  -o-user-select: none;
+  user-select: none;
 }
 
 #chartjs-tooltip {
-    opacity: 1;
-    position: absolute;
-    background: rgba(0, 0, 0, .7);
-    color: #fff;
-    border-radius: 3px;
-    -webkit-transition: all .1s ease;
-    -o-transition: all .1s ease;
-    transition: all .1s ease;
-    pointer-events: none;
-    -webkit-transform: translate(-50%, 0);
-    -ms-transform: translate(-50%, 0);
-    -o-transform: translate(-50%, 0);
-    transform: translate(-50%, 0);
+  opacity: 1;
+  position: absolute;
+  background-color: rgba(0, 0, 0, .7);
+  color: #fff;
+  border-radius: 3px;
+  pointer-events: none;
+  -webkit-transition: opacity .1s ease, -webkit-transform .1s ease;
+  transition: opacity .1s ease, -webkit-transform .1s ease;
+  -o-transition: opacity .1s ease, -o-transform .1s ease;
+  transition: opacity .1s ease, transform .1s ease;
+  transition: opacity .1s ease, transform .1s ease, -webkit-transform .1s ease, -o-transform .1s ease;
```

Comment:
this is not a duplicate property and I used the browserslistrc we used in Bootstrap 3.x which is what's used in this project. Although arguably there are some pretty ancient browsers included.